tests/v2/validation/rbac/rbac.go

```diff
@@ -0,0 +1,75 @@
+package rbac
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/namespaces"
+	"github.com/rancher/rancher/tests/framework/extensions/projects"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	provisioning "github.com/rancher/rancher/tests/v2/validation/provisioning"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+)
+
+//Helper function to create users
+func createUser(client *rancher.Client)(*management.User,error) {
+	enabled := true
+	var username = provisioning.AppendRandomString("testuser-")
+	user := &management.User{
+		Username: username,
+		Password: "rancherrancher123",
+		Name:     username,
+		Enabled:  &enabled,
+	}
+
+	newUser, err := users.CreateUserWithRole(client, user, "user")
+	if err != nil {
+	return newUser, err
+	}
+
+	newUser.Password = user.Password
+	return newUser, err
+}
+
+	
+//Gets the list of projects and return the names of the projects as a slice
+func GetListProjects(client *rancher.Client, clusterID string) (projectNames []string, err error) {
+	projectList, err := projects.GetProjectList(client, clusterID)
+	projectNames = make([]string, len(projectList.Data))
+
+	if err != nil {
+		return projectNames, err
+	}
+
+	for idx, project := range projectList.Data {
+		projectNames[idx] = project.Name
+	}
+	return projectNames, err
+}
+
+//(client *rancher.Client, clusterID string, listOpts metav1.ListOptions
+
+//Gets the list of namespaces and return the names of the namespaces as a slice
+func GetListNamespaces(client *rancher.Client, clusterID string) (namespace []string, err error) {
```

Comment:
Can we change this to `ListNamespace` instead? or `GetNamespace` ? Or is this how it's everywhere else in the framework ?

Reply:
It is GetNamespacesList - in general in the framework. But I changed it to GetNamespaces to be more generic